pkg/service/cluster/service.go

```diff
@@ -617,9 +617,12 @@ func SingleHostSessionConfigOption(host string) SessionConfigOption {
 			return errors.Wrapf(err, "fetch node (%s) info", host)
 		}
 		cqlAddr := ni.CQLAddr(host)
+		if ni.ClientEncryptionEnabled {
```

Comment:
I think here we should also consider cluster level ssl options like `ForceTLSDisabled` and `ForceNonSSLSessionPort`.

Reply:
Agree - I believe that it would be nicer to have just a single method that we use when we want to obtain valid CQL addr. It would need to accept mentioned SM flags as well. The current methods (`ni.CQLAddr`, `ni.CQLSSLAddr`) should be removed or un-exported so that we don't make this mistake in the future.
It would also result in de-duplicating some code.

Reply:
As you noticed, this fragment of code does not work correctly, as in some Scylla configurations it's possible to use SSL on a regula CQL port.

Reply:
Ok, thanks for your feedback,   I've replaced  CQLAddr and CQLSSLAddr with one function which takes into account cluster ssl setting

Reply:
Thanks!